private-templates-service/client/src/components/Common/ShareModal/ShareModal.tsx

```diff
@@ -21,11 +48,19 @@ class ShareModal extends React.Component<ShareModalProps> {
           <CenterPanelWrapper>
             <ShareLinkPanel>
               <SemiBoldText>Share with link</SemiBoldText>
-              <TextField readOnly={true} prefix={Config.redirectUri}>{"/preview/" + this.props.template.id + "/" + this.props.version}</TextField>
+              <LinkRow>
+                <TextFieldContainer>
+                  <TextField readOnly={true}
+                    prefix={Config.redirectUri}
+                    defaultValue={"/preview/" + this.props.template.id + "/" + this.props.templateVersion}
+                    width={100} />
+                </TextFieldContainer>
+                <Button iconProps={{ iconName: 'Copy' }} onClick={() => { onCopy(this.props) }}>
+                  Copy
+                </Button>
+              </LinkRow>
             </ShareLinkPanel>
-            <EmailPanel>
-              <SemiBoldText>Send to recipients</SemiBoldText>
-            </EmailPanel>
+            <ShareModalForm shareURL={Config.redirectUri + "/preview/" + this.props.template.id + "/" + this.props.templateVersion} templateVersion={this.props.templateVersion} />
```

Comment:
shareURL: the same code is up there for default value. Would it be better to extract it into a function to construct?